module-specific/hunters_mark-cast_mark.js

```diff
@@ -136,14 +137,31 @@ async function castSpell() {
 
 function setRangerSpell() {
   className = rangerName;
-  if (actor.items.find(i => i.name === `${rangerSpell}`)) {
-    spellName = rangerSpell;
+  if (actor.items.find(i => i.name === `${markSpell}`)) {
+    spellName = markSpell;
     castSpell();
   } else {
     ui.notifications.error(
-      "Selected actor does not have the " + rangerSpell + " spell."
+      "Selected actor does not have the " + markSpell + " spell."
     );
-    console.log("Selected actor does not have the " + rangerSpell + " spell.");
+    console.log("Selected actor does not have the " + markSpell + " spell.");
+    return null;
+  }
+}
+
+//If the class is paladin and has the hunter's mark spell, set the spell name
+//or say the spell doesn't exist.
+
+function setPaladinSpell() {
+  className = paladinName;
```

Comment:
These `className` variables don't seem to be used anywhere in this file.  If they aren't needed this function could be combined with the ranger function to avoid duplication.  But I don't know if this is a requirement for something else...